static/gsApp/views/seerAutomation/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,17 @@
+import useOrganization from 'sentry/utils/useOrganization';
+
+import SeerOnboardingLegacy from './onboardingLegacy';
+import SeerOnboardingSeatBased from './onboardingSeatBased';
+
+/**
+ * Depending on user's billing, will show either the legacy onboarding, or the newer, seat-based onboarding.
+ */
+export default function SeerOnboarding() {
+  const organization = useOrganization();
+
+  if (organization.features.includes('seat-based-seer-enabled')) {
```

Comment:
this'll need to be that new func we were just talking about. checking the sub (as it does) and the launch day flag.